extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java

```diff
@@ -13,10 +13,13 @@
 public class JibConfig {
 
     /**
-     * The base image to be used when a container image is being produced for the jar build
+     * The base image to be used when a container image is being produced for the jar build.
+     *
+     * When the application is built against Java 17 or higher, {@code TODO} is used as the default.
```

Comment:
You may want to update this, and mention the 2 defaults

Reply:
Good catch, fixed!